Reneiski_Homework1/ReneiskiSelenium11/Data/TestSettings.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.Extensions.Configuration;
+using ReneiskiSelenium11.Data.Enums;
+
+namespace ReneiskiSelenium11.Data
+{
+    public class TestSettings
+    {
+        public static Browsers Browser { get; set; }
```

Comment:
please correct properties and variables cases according to the coding standards (check please everywhere): 
![image](https://github.com/agrren/Reneiski_homeWork/assets/55654931/4207f0be-1a91-429a-97e9-74a7077153e1)
https://wiki.itechart-group.com/display/codstandarts/Coding+Standard+Visual+C+Sharp

Reply:
Fixed